index.js

```diff
@@ -1,13 +1,15 @@
 const express = require("express");
+const app = express();
```

Comment:
Don't create the app instance directly, Here firstly all the required libraries will be mentioned